sql/ok_result.go

```diff
@@ -0,0 +1,29 @@
+package sql
+
+// OkResult is a representation of the OK packet MySQL sends for non-select queries such as UPDATE, INSERT, etc. It
+// can be returned as the only element in the row for a Node that doesn't select anything.
+// See https://dev.mysql.com/doc/internals/en/packet-OK_Packet.html
+type OkResult struct {
+	RowsAffected uint64 // Number of rows affected by this operation
+	InsertID     uint64 // Inserted ID, if any, or -1 if not
+	Info         string // Human-readable status string for extra status info, echoed verbatim to clients.
```

Comment:
Don't love this.  Having it as a string makes it difficult to use any of that info in calling programs.  Might prefer it to be a io.Stringer.  Then a struct could be passed back and type assertion could be used to extract data from the Info.

Reply:
Good idea, fixed.